mypy/semanal.py

```diff
@@ -2128,6 +2128,17 @@ def visit_member_expr(self, expr: MemberExpr) -> None:
                 if full_name in obsolete_name_mapping:
                     self.fail("Module has no attribute %r (it's now called %r)" % (
                         expr.name, obsolete_name_mapping[full_name]), expr)
+        elif isinstance(base, RefExpr) and isinstance(base.node, TypeInfo):
```

Comment:
Cool, that makes sense.  Would you mind adding short comments to the two branches saying this?